e2e-tests/portal/cypress/e2e/journeys/maker/dashboard/deals/dashboard-empty.spec.js

```diff
@@ -10,7 +10,6 @@ context('Dashboard deals', () => {
 
   it('should render an empty dashboard', () => {
```

Comment:
This test doesnt actually seem to do anything - it doesnt test that the dashboard is empty (this would pass if there were deals there right?)

Reply:
No, the test will fail if deals are displayed.